### DISCUSSION TITLE
Total amount of storage used?

### DISCUSSION THREAD

**Discussion by eurstudent** [filecoin-project/community#185] [Q&A]
Hello all,

I am a Dutch student researching Filecoin and its adoption in the world. I want to display historical data of the amount of storage used compared to the amount of storage available. I have found using older website data on the filfox website the storage capacity of the filecoin network but was unable to find the amount of data that was being used. My question is: is there a website where I can find this kind of data or do you guys know a different way to quantify the usage of the network (perhaps messages on the blockchain?)
Thanks in advance!

**Comment by rjan90** [comment]
https://storage.filecoin.io/ lists some interesting storage-stats. Note that it lists only currently active deals, meaning that `Total storage deals`, etc, can be read as: `Total storage deals that are currently active`.

For seeing how many deals that have been proposed in total, you can check the counter on the f05-actor (storage-market actor) with: `lotus state read-state f05`, and look for the `NextID` number. Which is currently at 1 999 979 storage deals

**Reply by eurstudent** [reply]
Thanks for the quick reply!
This is very helpful. Unfortunately, I do not own a Linux device so am unable to run lotus. I am looking at the f05-actor in filfox and could find the total messages being equal to 2279566. Is this the same as the total storage deals?

**Reply by rjan90** [reply]
I see! You can see the total amount of Deals in upper left hand corner here: [https://filfox.info/en/deal](https://filfox.info/en/deal)

**Comment by jimpick** [comment]
I've made a visualization that shows the number of Filecoin deals on a daily or hourly basis:

https://observablehq.com/@jimpick/miner-report-published-deals-daily?collection=@jimpick/miner-report

For all of the current active deals, you can get a JSON file with all of the deals:

* https://marketdeals.s3.us-east-2.amazonaws.com/StateMarketDeals.json (warning, big file)

(or use the StateMarketDeals JSON-RPC API call on a Lotus node)
